apps/meteor/tests/e2e/omnichannel/omnichannel-send-pdf-transcript.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { faker } from '@faker-js/faker';
+import type { Page } from '@playwright/test';
+
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { OmnichannelLiveChat, HomeOmnichannel } from '../page-objects';
+import { test, expect } from '../utils/test';
+
+test.describe('omnichannel- export chat transcript as PDF', () => {
+    let poLiveChat: OmnichannelLiveChat;
+    let newUser: { email: string; name: string };
+
+    let agent: { page: Page; poHomeChannel: HomeOmnichannel };
+    test.beforeAll(async ({ api, browser }) => {
+        newUser = {
+            name: faker.person.firstName(),
+            email: faker.internet.email(),
+        };
+
+        // Set user user 1 as manager and agent
+        await api.post('/livechat/users/agent', { username: 'user1' });
+        await api.post('/livechat/users/manager', { username: 'user1' });
+
+        const { page } = await createAuxContext(browser, Users.user1);
+        agent = { page, poHomeChannel: new HomeOmnichannel(page) };
+    });
+    test.beforeEach(async ({ page, api }) => {
+        poLiveChat = new OmnichannelLiveChat(page, api);
+    });
+
+    test.afterAll(async ({ api }) => {
+        await api.delete('/livechat/users/agent/user1');
+        await api.delete('/livechat/users/manager/user1');
+        await agent.page.close();
+    });
+
+    test('Export PDF transcript', async ({ page }) => {
+        await test.step('Expect send a message as a visitor', async () => {
+            await page.goto('/livechat');
+            await poLiveChat.openLiveChat();
+            await poLiveChat.sendMessage(newUser, false);
+            await poLiveChat.onlineAgentMessage.type('this_a_test_message_from_visitor');
+            await poLiveChat.btnSendMessageToOnlineAgent.click();
+        });
+
+        await test.step('Expect to have 1 omnichannel assigned to agent 1', async () => {
+            await new Promise(resolve => setTimeout(resolve, 5000));
+            await agent.poHomeChannel.sidenav.openChat(newUser.name);
+        });
+
+        await test.step('Expect to be not able send transcript as PDF', async () => {
+            test.skip(!IS_EE, 'Enterprise Only');
```

Comment:
This whole test seems to be EE only. Wouldn't it be better to skip everything when on CE?
If so, I recommend adding the skip at the beginning of the file.